{
	"candidate": "CVE-2024-6237",
	"description": "A flaw was found in the 389 Directory Server. This flaw allows an unauthenticated user to cause a systematic server crash while sending a specific extended search request, leading to a denial of service.",
	"priority": "medium",
	"cvss": {
		"nvd": " CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H [6.5 MEDIUM]"
	},
	"references": [
		"https://www.cve.org/CVERecord?id=CVE-2024-6237",
		"https://github.com/389ds/389-ds-base/issues/5989",
		"https://github.com/389ds/389-ds-base/commit/e8dd583685e6143f2027f97569de4cc45ba46e14 (389-ds-base-2.4.5)",
		"https://access.redhat.com/security/cve/CVE-2024-6237",
		"https://bugzilla.redhat.com/show_bug.cgi?id=2293579"
	],
	"public_date": "2024-07-09 17:15:00 UTC",
	"packages": {
		"devel": {
			"389-ds-base": {
				"name": "389-ds-base",
				"status": "not-affected"
			}
		},
		"esm-apps/bionic": {
			"389-ds-base": {
				"name": "389-ds-base",
				"status": "needs-triage"
			}
		},
		"esm-apps/focal": {
			"389-ds-base": {
				"name": "389-ds-base",
				"status": "needs-triage"
			}
		},
		"esm-apps/jammy": {
			"389-ds-base": {
				"name": "389-ds-base",
				"status": "needs-triage"
			}
		},
		"esm-apps/noble": {
			"389-ds-base": {
				"name": "389-ds-base",
				"status": "not-affected"
			}
		},
		"esm-apps/xenial": {
			"389-ds-base": {
				"name": "389-ds-base",
				"status": "needs-triage"
			}
		},
		"focal": {
			"389-ds-base": {
				"name": "389-ds-base",
				"status": "needs-triage"
			}
		},
		"jammy": {
			"389-ds-base": {
				"name": "389-ds-base",
				"status": "needs-triage"
			}
		},
		"mantic": {
			"389-ds-base": {
				"name": "389-ds-base",
				"status": "ignored",
				"note": "end of life, was needs-triage"
			}
		},
		"noble": {
			"389-ds-base": {
				"name": "389-ds-base",
				"status": "not-affected",
				"note": "2.4.5+dfsg1-1"
			}
		},
		"upstream": {
			"389-ds-base": {
				"name": "389-ds-base",
				"status": "released",
				"note": "2.4.5+dfsg1-1"
			}
		}
	}
}
